lib/LMSManagers/LMSConfigManager.php

```diff
@@ -134,12 +138,12 @@ public function GetConfigDefaultType($section, $var)
         return $type;
     }
 
-    public function CheckOption($type, $value)
+    public function CheckOption($var, $value, $section, $type)
```

Comment:
A może jednak przywrócić starą definicję, żeby w ogóle nie ruszać interfejsu, a przygotować drugą metodę, która będzie miała to co tu robisz no i żeby nie duplikować kodu wywołać z CheckOption tą nową metodę z odpowiednimi parametrami? Dzięki temu nie naruszy się interfejsu i kod, który już zakłada taki interfejs w klasie nadal będzie działał.